src/shared/util.js

```diff
@@ -1097,6 +1097,35 @@ const FontRenderOps = {
   TRANSLATE: 8,
 };
 
+// TODO: Remove this once `Uint8Array.prototype.toHex` is generally available.
+function toHexUtil(arr) {
+  if (Uint8Array.prototype.toHex) {
```

Comment:
Could we avoid this check in the m-c case ?

Reply:
Yes, but I don't believe that that would be a good idea just yet unfortunately.

Please note that this is really new JavaScript functionality, which can be disabled with a preference (see below), hence if this leads to breakage in real-world websites it's possible that the pref will be disabled *temporarily* and the Firefox PDF Viewer would thus be broken by that.

https://searchfox.org/mozilla-central/rev/a547d790fa7facb8d0addb9197797dc52b31e8f2/modules/libpref/init/StaticPrefList.yaml#8119-8124

Reply:
Oh my bad, I thought it wasn't implemented in Firefox yet.

Reply:
FYI: It'll be available by default from Firefox 133, see https://bugzilla.mozilla.org/show_bug.cgi?id=1917885.